### PR TITLE
Update chat RFC status to ACCEPTED

### PR DESCRIPTION
**Pull request by mbestavros**
Per @npmccallum, this should be a place to debate and discuss specific acceptance criteria for our chat RFC.

### CODE REVIEW

**Review by haraldh — DISMISSED** [review]

File: 00000-migrate-enarx-public-chat/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 00000-migrate-enarx-public-chat
 - Authors: Mark Bestavros, Axel Simon
-- Status: [DEMONSTRATED](/README.md#demonstrated)
+- Status: [ACCEPTED](/README.md#acepted)
```
Comment (connorkuehl) [inline 00000-migrate-enarx-public-chat/README.md]: Typo: ~~acepted~~ -> accepted

**Review by connorkuehl — CHANGES_REQUESTED** [review]

File: 00000-migrate-enarx-public-chat/README.md (hunk 1 above)
Comment (mbestavros) [reply 00000-migrate-enarx-public-chat/README.md]: Fixed. Thanks!

**Review by lkatalin — COMMENTED** [review]
Is this PR only for the chat program itself and not the RFC about the chat program? The RFC about the chat program needs to update the chat link for Rocket.chat.

But re: the program itself, I have accepted it FWIW.

**Comment by npmccallum** [comment]
> Is this PR only for the chat program itself and not the RFC about the chat program? The RFC about the chat program needs to update the chat link for Rocket.chat.
> 
> But re: the program itself, I have accepted it FWIW.

@mbestavros Please fix this in a separate PR.

**Review by MikeCamel — APPROVED** [review]

**Review by connorkuehl — APPROVED** [review]

**Review by lkatalin — APPROVED** [review]

**Review by axelsimon — APPROVED** [review]

**Review by ueno — APPROVED** [review]

**Comment by npmccallum** [comment]
Seeing that we have consensus, the status change is approved.

**Pull request merged**